azure-pipelines.yml

```diff
@@ -64,7 +64,7 @@ stages:
         sudo service redis-server start
 
         # LIBYANG
-        sudo dpkg -i ../target/debs/bookworm/libyang*1.0.73*.deb
+        sudo dpkg -i ../target/debs/bookworm/libyang*3.*.deb
```

Comment:
Pipeline build for this PR is failing with error:
> dpkg: error: cannot access archive '../target/debs/bookworm/libyang*3.*.deb': No such file or directory

Is the deb path correct? Please fix this so that all CVL tests are exercised and libyang usages are validated.

Reply:
@sachinholla since https://github.com/sonic-net/sonic-buildimage/pull/21679 is not merged, the debian packages are not uploaded as artifacts that can be retrieved at this point in time.  The azure pipelines therefore cannot be used for validation.  It requires manual compilation and validation in a local environment, which requires that you've installed the 'devel' branch of [libyang](https://github.com/CESNET/libyang/tree/devel).  

I've submitted and have had accepted patches I created that sonic depends on in libyang, but they are not yet in a release version ... this PR does depend on those. (https://github.com/sonic-net/sonic-buildimage/pull/21679 does include said patches already).

Reply:
@bradh352 sachinholla is the expert of the sonic-mgmt-common repository. You can seek help from him.

Reply:
Hi Brad, understood, I missed your previous comments.. You can can run the same CVL tests locally using `make && make cvl-test` from your sonic-0mgmt-common repo. Details in my response to your [UMF WG post](https://lists.sonicfoundation.dev/g/umf-working-group/message/87).